### ISSUE TITLE
🐛 Biome encountered an unexpected error

### ISSUE THREAD

**Issue by marmikdesai** [biomejs/biome#4065]
### Environment information

```block
2024-09-24 16:48:50.133 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:75:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.


2024-09-24 16:48:50.622 [info] [Error - 16:48:50] Biome language server closed
```

### What happened?

While I start visual studio.

### Expected result

It should not crash

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by markmansur** [comment]
Seeing the same issue

**Comment by ematipico** [comment]
To create a reproduction, you can use our new CLI tool, which should help you to create a very quick repository with what you need: `npm create @biomejs/biome-reproduction`

**Comment by gnosticdev** [comment]
I had this error in VSCode as well, but was able to fix:

- remove old/bad rules from the config file that may have been removed in the last update.
- run `biome check .` in project root with the project's `biome.json` config file included, this will cause it to throw on invalid properties.

I removed the invalid key - `noDuplicateJsonKeys` in my case, then restarted the LSP server and it finally went away.

hope it helps anyone else having the issue.

**Comment by a613** [comment]
Why closed? Linked issue is marked a dupe of this so this should be the canonical issue for "You must have at least one workspace".

No indication that this is always caused by invalid rules in the biome config.

**Comment by dyc3** [comment]
It was auto closed because nobody's submitted a repro yet. We can't do anything without a valid repro.

**Comment by pjamrozowicz** [comment]
I had the same error as above. For me it was caused by using the latest VSCode plugin + new biome config + **old** biome installation. 

I simply forgot to run `npm install`. Once I did and restarted biome - it works great.
